docs/how-to/configure-dns.md

```diff
@@ -0,0 +1,88 @@
+As part of the go-live process for your site you will need to update your DNS
+records to point to your Fleet installation. For new sites this is part of the
+setup process. For existing sites being migrated to Fleet, this is one of the
+final, critical steps that you'll take.
+
+This page assumes a basic understanding of DNS concepts and common record types
+(A, CNAME, etc), and basic Fleet terminology (environments, releases).
+
+
+Our example Fleet
+----
+
+For the purposes of explanation, we will use a fictional retailer named **Best
+Pants**. Best Pants has a Fleet named `bestpants`, and two environments: `prod`
+and `dev`. Best Pants uses bestpants.com for their website.
+
+
+The records needed
+----
+
+Each environment has automatically-created internal DNS names. Using the `prod`
+environment from *Best Pants* as our example, they are:
+
+```
+   admin.prod.bestpants.f.nchr.io
+adminssh.prod.bestpants.f.nchr.io
+    solr.prod.bestpants.f.nchr.io
+     www.prod.bestpants.f.nchr.io
+```
+
+For the dev environment, there will likewise be `www.dev.bestpants.f.nchr.io`,
+`admin.dev.bestpants.f.nchr.io`, etc.
+
+
+Mapping real DNS names to internal DNS records
+----
+
+In order to be useful and accessible, hostnames in the bestpants.com domain are
+mapped to the internal names with `CNAME` DNS records:
+
+```
+   admin.bestpants.com  ->     admin.prod.bestpants.f.nchr.io
+adminssh.bestpants.com  ->  adminssh.prod.bestpants.f.nchr.io
+    solr.bestpants.com  ->      solr.prod.bestpants.f.nchr.io
```

Comment:
probably leave out solr since it won't be public facing. adminssh is debatable. www and admin are the only endpoints that should be public facing.